### PR TITLE
skip linkifyBranchRefs() if fork has been removed

### PR DESCRIPTION
**Pull request by Ma27**
Minor bug I've observed with the branch ref linkifications if the repository has been removed.
This change avoids linkifying a branch if the repository is unknown as this would result in broken links.

### CODE REVIEW

**Comment by sindresorhus** [comment]
LGTM

**Comment by paulmolluzzo** [comment]
Small issue: Even though we're not adding a link we still underline the text on `:hover`. This is because the CSS isn't specific enough here: https://github.com/sindresorhus/refined-github/blob/8ade0dd/extension/content.css#L156-160

Could you add the `a` tag in the selectors like this? (Unless there's a better solution):

```css
/* add hover underline for `content.js` linkified branch refs in pull requests */
a .commit-ref:hover,
a .commit-ref:hover span {
	text-decoration: underline !important;
}
```

**Comment by Ma27** [comment]
good point @paulmolluzzo, will have a look at it in the next time

**Comment by Ma27** [comment]
done @paulmolluzzo @sindresorhus

File: extension/content.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ const getUsername = () => $('meta[name="user-login"]').attr('content');
 
 function linkifyBranchRefs() {
 	$('.commit-ref').each((i, el) => {
+		if ($(el).children().eq(0).text() === 'unknown repository') {
```
Comment (DrewML) [inline extension/content.js]: Let's cache the value of `$(el)` so we don't have to hit the DOM twice for the same element.

`const $el = $(el);`

**Comment by sindresorhus** [comment]
@Ma27 Can you update the pull request regarding https://github.com/sindresorhus/refined-github/pull/253#discussion_r69395442?

**Pull request merged**